frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/unload-arriving-vehicles/simulated-region-overview-behavior-unload-arriving-vehicles.component.ts

```diff
@@ -8,4 +23,93 @@ import { Component } from '@angular/core';
         './simulated-region-overview-behavior-unload-arriving-vehicles.component.scss',
     ],
 })
-export class SimulatedRegionOverviewBehaviorUnloadArrivingVehiclesComponent {}
+export class SimulatedRegionOverviewBehaviorUnloadArrivingVehiclesComponent
+    implements OnInit
+{
+    @Input()
+    simulatedRegionId!: UUID;
+
+    @Input()
+    behaviorId!: UUID;
+
+    unloadDuration$?: Observable<number>;
+    vehiclesStatus$?: Observable<VehicleUnloadStatus[]>;
+
+    constructor(
+        private readonly exerciseService: ExerciseService,
+        public readonly store: Store<AppState>
+    ) {}
+
+    ngOnInit(): void {
```

Comment:
I think it could be handy to move some of the rather generic selectors in this function (e.g. `behaviorStateSelector` and `activityStateSelector`) to `frontend/src/app/state/application/selectors/exercise.selectors.ts` so they can be reused and using rxjs pipes for the more specific ones (e.g. for `unloadDuration$`).